subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/ProgressPlainLoggingIntegrationTest.groovy

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.logging
+
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+
+class ProgressPlainLoggingIntegrationTest extends AbstractIntegrationSpec {
```

Comment:
Do we also need to test interleaved tasks? like https://github.com/gradle/gradle/blob/656eec72102b23637285a23c966e08e015862e15/subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/BasicGroupedTaskLoggingFunctionalSpec.groovy#L119-L119